src/components/Select.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/react';
+
+const Select = ({ disabled, ...props }) => (
+  <div
+    css={css`
+      height: 28px;
+      display: grid;
+      grid-template-areas: 'select';
+      align-items: center;
+      width: 100%;
+      min-width: 15ch;
+      border: 1px solid var(--color-neutrals-500);
+      border-radius: 0.25rem;
+      padding: 0.25rem 0.5rem;
+      cursor: pointer;
+      position: relative;
+
+      ${disabled &&
+      css`
+        cursor: not-allowed;
+        opacity: 0.5;
+
+        .light-mode & {
+          background-color: var(--color-neutrals-100);
+        }
+      `}
+
+      .dark-mode & {
+        border: 1px solid var(--color-dark-300);
+        background-color: var(--color-dark-300);
+      }
+
+      &::after {
+        content: '';
+        grid-area: select;
+        justify-self: end;
+        width: 0.5rem;
+        height: 0.25rem;
+        clip-path: polygon(100% 0%, 0 0%, 50% 100%);
```

Comment:
Out of curiosity, what are you trying to achieve here? [css-shapes](https://caniuse.com/css-shapes) is semi-supported.

Reply:
oh we use this for the data dictionary filter select dropdowns as well. we create a custom select dropdown `div` that looks like the default, but this way we have a bit more control over styling / arrow.

Reply:
that said, we could probably achieve most of what we want without the extra `div`, but the caret is a little thicker (in Chrome anyway) and may vary across browsers.

@zstix i'm kinda torn. should we ditch some of the extra-ness here and go for simplicity OR follow that pattern that gives us a little extra control (seemingly mostly for the caret)